rbac/rbac/settings.py

```diff
@@ -353,6 +353,7 @@
 V2_MIGRATION_RESOURCE_EXCLUDE_LIST = ENVIRONMENT.get_value("V2_MIGRATION_RESOURCE_EXCLUDE_LIST", default="").split(",")
 V2_BOOTSTRAP_TENANT = ENVIRONMENT.bool("V2_BOOTSTRAP_TENANT", default=False)
 V1_BOOTSTRAP_ADD_USER_ID = ENVIRONMENT.bool("V1_BOOTSTRAP_ADD_USER_ID", default=False)
+LOG_REPLICATION = ENVIRONMENT.bool("LOG_REPLICATION", default=False)
```

Comment:
We could consider an "aggregate replicator" which passed the event to multiple implementations, and then optionally using this with the logging replicator plus another impl, as an elegant way to add logging to any replicator, but I'm not sure if it's worth it. Or if it is, maybe this can be its own small PR.

Reply:
I removed but such simple attaching to outbox replication was definitely worth to local debugging without any other changes :)